src/youtube-player/README.md

```diff
@@ -58,3 +58,65 @@ import {YouTubePlayer, YOUTUBE_PLAYER_CONFIG} from '@angular/youtube-player';
 })
 export class YourApp {}
 ```
+
+## Loading behavior
+By default the `<youtube-player/>` will show a placeholder element instead of loading the API
+up-front until the user interacts with it. This speeds up the initial render of the page by not
+loading unnecessary JavaScript for a video that might not be played. Once the user clicks on the
+video, the API will be loaded and the placeholder will be swapped out with the actual video.
```

Comment:
Is it worth mentioning that the placeholder:

- has a lower quality of the thumbnail (as I can see in the visual diff?)
- does not show the title of the video? (or should we show that as part of the placeholder)

Reply:
Yeah good point, I'll call them out in the doc. I think that generally users won't notice the difference since they won't get to see the YouTube placeholder.

Reply:
Actually it seems like we can use the `maxresdefault.jpg` placeholder instead. When I was testing earlier the highest-quality webp videos didn't have placeholders, but the jpg ones do. I'll switch it to the high quality one and we can revisit it later if it's problematic.